project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
-  // constraints and models datatypes in ways that are incompatible with our test expectations.
-  // liquibase-core 4.0.0 did not have either of those problems but produced tons of strange warnings at runtime
-  // similar in form to this: https://github.com/liquibase/liquibase/issues/1294
-  // Pinning Liquibase version for the time being.
-  private val liquibaseV = "3.6.3" // scala-steward:off
-  private val logbackV = "1.2.3"
-  private val lz4JavaV = "1.7.1"
-  private val mariadbV = "2.7.2"
-  private val metrics3ScalaV = "4.0.0" // https://github.com/erikvanoosten/metrics-scala/tree/f733e26#download-4x
+  private val liquibaseV = "4.4.0" // 4.4.1+ needs https://github.com/liquibase/liquibase/pull/2001
+  private val logbackV = "1.2.5"
+  private val lz4JavaV = "1.8.0"
+  private val mariadbV = "2.7.4"
+  /*
+  The StatsD reporter for DropWizard's (Code Hale's) Metrics 3.x still works with Metrics 4.x.
+  Still would be great to move to Prometheus / OpenCensus
+   */
+  private val metrics4ScalaV = "4.1.19"
   private val metrics3StatsdV = "4.2.0"
   private val mockFtpServerV = "2.8.0"
-  private val mockserverNettyV = "5.5.4" // scala-steward:off (CROM-6582)
-  private val mouseV = "1.0.0"
-  private val mysqlV = "8.0.22" // scala-steward:off (CROM-6772: DateTime/String conversion and connection string updates required)
-  private val nettyV = "4.1.46.Final"
-  private val owlApiV = "5.1.16" // scala-steward:off (CROM-6677)
+  private val mockserverNettyV = "5.11.2"
+  private val mouseV = "1.0.4"
+  private val mysqlV = "8.0.26"
+  private val nettyV = "4.1.66.Final"
+  private val owlApiV = "5.1.19"
   private val paradiseV = "2.1.1"
   private val pegdownV = "1.6.0"
-  // For org.postgresql:postgresql 42.2.6 - 42.2.14:
-  // java.lang.NoSuchFieldException: m_mesgParts in KeyValueSpec "fail if one of the inserts fails"
-  private val postgresV = "42.2.5" // scala-steward:off
-  private val rdf4jV = "2.4.2"
-  private val refinedV = "0.9.22"
+  private val postgresV = "42.2.23"
+  private val pprintV = "0.6.6"
+  private val rdf4jV = "3.7.1"
+  private val refinedV = "0.9.27"
   private val rhinoV = "1.7.13"
+  private val scalaCollectionCompatV = "2.5.0"
   private val scalaGraphV = "1.13.1"
-  private val scalaLoggingV = "3.9.3"
+  private val scalaLoggingV = "3.9.4"
   private val scalaPoolV = "0.4.3"
-  private val scalacheckV = "1.15.3"
-  private val scalacticV = "3.2.6"
+  private val scalacheckV = "1.15.4"
+  private val scalacticV = "3.2.9"
   private val scalameterV = "0.19"
   private val scalamockV = "5.1.0"
-  private val scalatestV = "3.2.6"
+  private val scalatestV = "3.2.9"
   private val scalatestPlusMockitoV = "1.0.0-M2"
-  private val scalazV = "7.3.3"
+  private val scalazV = "7.3.5"
   private val scoptV = "4.0.1"
-  private val sentryLogbackV = "1.7.30" // scala-steward:off (CROM-6640)
-  private val shapelessV = "2.3.3"
+  private val sentryLogbackV = "5.1.2"
```

Comment:
This feels like a huge jump and I wouldn't even know where to start, so... just saying that I'm leaving this one alone and assuming that any problems would be caught by compilation, tests, or general operations.